addins/Cake.Issues.Testing.yml

```diff
@@ -2,7 +2,7 @@ Name: Cake.Issues.Testing
 NuGet: Cake.Issues.Testing
 Assemblies:
 - "/**/Cake.Issues.Testing.dll"
-Repository: https://github.com/cake-contrib/Cake.Issues.Testing
+Repository: http://cake-contrib.github.io/Cake.Issues.Website/
```

Comment:
Is there a reason that this switched from https to http?

Reply:
probably because nuspec now is
```xml
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
  <metadata>
    <id>Cake.Issues.Testing</id>
    <version>0.7.0</version>
    <title>Cake.Issues.Testing</title>
    <authors>BBT Software AG and contributors</authors>
    <owners>bbtsoftware,pascalberger,cake-contrib</owners>
    <requireLicenseAcceptance>false</requireLicenseAcceptance>
    <license type="expression">MIT</license>
    <projectUrl>http://cake-contrib.github.io/Cake.Issues.Website/</projectUrl>
    <iconUrl>https://cdn.jsdelivr.net/gh/cake-contrib/graphics@a5cf0f881c390650144b2243ae551d5b9f836196/png/cake-contrib-medium.png</iconUrl>
    <description>Common helpers for testing add-ins based on Cake.Issues</description>
    <summary>Common helpers for testing add-ins based on Cake.Issues</summary>
    <releaseNotes>https://github.com/cake-contrib/Cake.Issues/releases/tag/0.7.0</releaseNotes>
    <copyright>Copyright © BBT Software AG and contributors</copyright>
    <tags>Cake Script Cake-Issues Issues Testing</tags>
    <repository type="git" url="https://github.com/cake-contrib/Cake.Issues" />
  </metadata>
</package>
```

Even though this probably should be corrected to `https` perhaps we should use `repository` tag if present?

Reply:
The AddinDiscoverer enforces HTTPS when the domain name is `github.com` only. I will modify the logic to also include `github.io`. 

Any other domain names you can think of I should include?